### ISSUE TITLE
Possibility of resizing the size of the call stack.

### ISSUE THREAD

**Issue by Pedromdsn** [oven-sh/bun#5031]
### What is the problem this feature would solve?

I have a problem in which the only solution (that I could think of) would be a somewhat extensive recursive function in some cases, and with the default call stack I quickly run out of space.

![image](https://github.com/oven-sh/bun/assets/57453260/390f47f9-59e3-4613-a1a1-1df4798db5e2)

### What is the feature you are proposing to solve the problem?

With this, the BUN user should be able to resize the size of their application's call stack at will (and risk).

**Comment by Braayy** [comment]
I second this, there should be an flag that can be used to increase/decrease the maximum call stack size.

**Comment by Electroid** [comment]
Does Node.js allow you to customize this?

**Comment by Braayy** [comment]
Yes! Using the `--stack-size` flag. eg. `node --stack-size=1000 index.js`

**Comment by MulverineX** [comment]
I'm curious if anyone has investigated implementing this?
